infrastructure/code/code.go

```diff
@@ -116,10 +116,26 @@ func (sc *Scanner) SupportedCommands() []snyk.CommandName {
 }
 
 func (sc *Scanner) Scan(ctx context.Context, path string, folderPath string) (issues []snyk.Issue, err error) {
-	if !sc.isSastEnabled() {
+	sastResponse, err := sc.SnykApiClient.SastSettings()
+	method := "Scan"
+
+	if err != nil {
+		log.Error().Err(err).Str("method", method).Msg("couldn't get sast enablement")
+		sc.errorReporter.CaptureError(err)
+		return issues, errors.New("couldn't get sast enablement")
+	}
+
+	if !sc.isSastEnabled(sastResponse) {
 		return issues, errors.New("SAST is not enabled")
 	}
 
+	if sc.isLocalEngineEnabled(sastResponse) {
+		isCodeApiUpdated := sc.updateCodeApiLocalEngine(sastResponse)
+		if !isCodeApiUpdated {
```

Comment:
Should we log the local engine url in the logs (info level),  in case we need to debug problems with it from customers?

Reply:
@michelkaporin In general this url is available in the org

Reply:
I wouldn't log it on info level. But up to you :)

Reply:
@shirlupo sure. Just wondering if we get a customer ticket saying Code analysis isn't working and we see the org is using LCE, can we figure out what's the exact issue inside of IDE?

Reply:
@michelkaporin Honestly hard to say without mentioning which kind of problem, for example, if the problem is creating the bundle so we can see it logs codeApi URL (in this case it will include within the LCE Url)

the logic should not change inside the IDE from the regular logic you know just pointing to different endpoints.

but for clarity - the LCE url is configured in the admin panel of that org